### PR TITLE
docs: update gRPC related docs

### LINKED ISSUE #264: Improve the gRPC protocol reference
https://docs.greptime.com/user-guide/supported-protocols/grpc

It's too simple and doesn't contain any useful information. We should describe the protocol in detail.

### PR DESCRIPTION
**Pull request by MichaelScofield**
close #264

### CODE REVIEW

**Review by xtang — APPROVED** [review]
LGTM

File: docs/user-guide/supported-protocols/grpc.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+Please note that currently service `GreptimeDatabase` can only handle insert requests. For query and DDL requests, you
+can use our gRPC service that is built on top
+of [Apache Arrow Flight](https://arrow.apache.org/docs/format/Flight.html). Arrow Flight is an RPC framework for
+high-performance data services based on Arrow data. You can refer to its official site for more details.
+
+## SDK
+
+We offer the following officially supported SDK:
+
+- [Java](../java-sdk)
```
Comment (xtang) [inline docs/user-guide/supported-protocols/grpc.md:36]:
```suggestion
- [Java](../java-sdk/java-sdk)
```

**Review by xtang — CHANGES_REQUESTED** [review]
fixed the java_sdk path

**Comment by MichaelScofield** [comment]
@xtang PTAL

**Review by xtang — APPROVED** [review]
LGTM

**Comment by xtang** [comment]
@killme2008 PTAL

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**